### DISCUSSION TITLE
Release schedule for crosswalk

### DISCUSSION THREAD

**Discussion by grgmiller** [USEPA/camd-eia-crosswalk#36] [General] [answered]
First off, thank you for enabling discussions on this repository! I wanted to ask a question for the maintainers to better understand how often we should expect new releases of the crosswalk, and what the expected lag will be for incorporating new data going forward. 

I understand that currently, the most recent version of the crosswalk is based on 2018 data, so I wanted to try and better understand what goes into incorporating more recent data - for example, with the most recent v0.3 release, what was limiting the ability to incorporate 2021 EIA-860 data? My understanding from skimming the R code and documentation is that many of the crosswalking steps are largely automated. I know that some data has to be manually crosswalked, but would it be possible to run the R script using more recent data and then supplement that over time with additional manual review?

On a related note, I was wondering how frequently EPA plans to release updates to the crosswalk, and on what kind of lag once the input data is released. For projects such as the [Open Grid Emissions Initiative](https://github.com/singularity-energy/open-grid-emissions), we would like to try and release 2021 data as soon after the final EIA-923 data for 2021 is published, and having a version of the crosswalk updated based on 2021 associations would be helpful.

I know that your team has many other priorities, and I appreciate this resource being made available at all, but additional clarity about what factors affect the frequency and freshness of the crosswalk releases would be helpful for users of the crosswalk. Thank you!

**Comment by j-tafoya** [comment, marked as answer]
Hi @grgmiller, 
Thanks so much for your active participation in this repo! Many of your questions (here and in your PR comments) may be answered by our planned release of another major update (v1.0) sometime next year, which will include updates to the script to make it easier to use with other programming languages as well as updates for more recent data. This includes updates that incorporate all the latest eGRID plant id crosswalk entries. The current v0.3 release only includes plant id matches up to eGRID 2018. Like you mentioned, if you need a more recent crosswalk sooner, you are welcome to change the data year in the R script, but we have not performed any QA/QC or manual matches for years after 2018. We welcome any insights you might gather from running the script for any recent data years and any pull requests that improve the script!

**Reply by grgmiller** [reply]
Hi Johnathan, thanks for the update - this is really helpful!
